dev/how-to/maintain/backup-and-restore/br.md

```diff
@@ -0,0 +1,376 @@
+---
+title: Use BR to Backup and Restore Cluster Data
+summary: Learn how to backup and restore the data of the TiDB cluster using BR.
+category: how-to
+---
+
+# Use BR to Backup and Restore Cluster Data
+
+Backup & Restore (BR) is a command-line tool for distributed backup and restoration of the TiDB cluster data. Compared with [`mydumper`/`loader`](/dev/how-to/maintain/backup-and-restore/mydumper-loader.md), BR is more suitable for scenarios of huge data volume. This document describes the command line, offers detailed use cases, best practices, usage restrictions, and introduces the working principle of BR.
+
+## Command-line description
+
+A `br` command consists of sub-commands, options and parameters. The sub-command is the characters without `-` or `--`. The option is the characters that start with `-` or `--`. The parameter is the characters that immediately follow and pass to the sub-command or the option.
+
+This is a complete `br` command:
+
+`br backup full --pd "${PDIP}:2379" -s "local:///tmp/backup"`
+
+In this command,
+
+* `backup` is the sub-command of `br`.
+* `full` is the sub-command of `backup`.
+* `-s` or `--storage` specifies the path in which the backup files are stored.
+* `"local:///tmp/backup"` is the parameter of `-s`. `/tmp/backup` is the path in the local disk in which the backup files are stored.
+* `--pd` is the Placement Driver (PD) service address.
+* `"${PDIP}:2379"` is the parameter of `--pd`.
+
+### Sub-commands
+
+A `br` command consists of multiple layers of sub-commands. Currently, BR has the following three sub-commands:
+
+* `br backup` is used to backup the data of the TiDB cluster.
+* `br restore` is used to restore the data of the TiDB cluster.
+* `br version` is used to check the version of BR.
+
+Each of the above three sub-commands might include the following three sub-commands to specify the scope of an operation:
+
+* `full` is used to backup or restore all the cluster data.
+* `db` is used to restore the specified database of the cluster.
+* `table` is used to backup or restore a single table in the specified database of the cluster.
+
+### Common options
+
+* `--pd` is for connection, specifying the PD server address. For example, `"${PDIP}:2379"`.
+* `-h`/`--help` is used to get help on all sub-commands. For example, `br backup --help`.
+* `--ca` specifies the path to the trusted CA certificate in the PEM format.
+* `--cert` specifies the path to the SSL certificate in the PEM format.
+* `--key` specifies the path to the SSL certificate key in the PEM format.
+* `--status-addr` specifies the listening address through which BR provides statistics to Prometheus.
```

Comment:
Consider converting this section into a table with two columns: **Option** and **Description**.